examples/Example/src/prot_snap/index.tsx

```diff
@@ -0,0 +1,347 @@
+import React, { useState } from 'react';
+import { Alert, StyleSheet, useWindowDimensions } from 'react-native';
+import { GestureMonitor, Gesture } from 'react-native-gesture-handler';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  runOnJS,
+} from 'react-native-reanimated';
+import { useAnimatedGesture } from '../useAnimatedGesture';
+
+const filters = ['red', 'green', 'blue', 'yellow', 'orange', 'cyan'];
+const MAX_VIDEO_DURATION = 60000;
+
+export default function Home() {
+  const filter = useSharedValue(0);
+  const filterOffset = useSharedValue(0);
+  const zoom = useSharedValue(1);
+  const [scale, setScale] = useState(1);
+  const [selectedFilter, setSelectedFilter] = useState(0);
+  const [recording, setRecording] = useState(false);
+  const [remainingTime, setRemainingTime] = useState(MAX_VIDEO_DURATION);
+  const [recordingInterval, setRecordingInterval] = useState(-1);
```

Comment:
`Interval` here is not precise in this context. I think `recordingIntervalHandle` or something more associated with `setInterval` function than word "interval" would be better here.

Reply:
Fixed in a480cf06cea6359d38ee763245440f50d6c3b2b2